src/Common/ya.make

```diff
@@ -0,0 +1,109 @@
+LIBRARY()
+
+ADDINCL (
+    GLOBAL clickhouse/src
+    contrib/libs/libcpuid
+    contrib/libs/libunwind/include
+    GLOBAL contrib/restricted/ryu
+)
+
+PEERDIR(
+    clickhouse/base/common
+    clickhouse/base/pcg-random
+    clickhouse/base/widechar_width
+    contrib/libs/libcpuid/libcpuid
+    contrib/libs/openssl
+    contrib/libs/re2
+    contrib/restricted/ryu
+)
+
+# TODO: stub for config_version.h
+CFLAGS (GLOBAL -DDBMS_NAME=\"ClickHouse\")
+CFLAGS (GLOBAL -DDBMS_VERSION_MAJOR=0)
+CFLAGS (GLOBAL -DDBMS_VERSION_MINOR=0)
+CFLAGS (GLOBAL -DDBMS_VERSION_PATCH=0)
+CFLAGS (GLOBAL -DVERSION_FULL=\"ClickHouse\")
+CFLAGS (GLOBAL -DVERSION_INTEGER=0)
+CFLAGS (GLOBAL -DVERSION_NAME=\"ClickHouse\")
+CFLAGS (GLOBAL -DVERSION_OFFICIAL=\"\\\(arcadia\\\)\")
```

Comment:
Ok but not sure that we need all these backslashes...

Reply:
I'm not sure too. Anyway I have to provide real versions later.